expose aliases

EtienneLt opened the pull request:
Signed-off-by: Etienne LESOT <etienne.lesot@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
#526

EtienneLt committed:
expose aliases

Signed-off-by: Etienne LESOT <etienne.lesot@rte-france.com>

EtienneLt commented:
doc is missing

sylvlecl commented on java/src/main/java/com/powsybl/dataframe/network/adders/AliasDataframeAdder.java:
```diff
@@ -0,0 +1,91 @@
+            }
+            if (aliasTypes != null) {
+                String type = aliasTypes.get(row);
+                if (type != null) {
```
Here it will never be null in practice, I think we should check for empty instead

sylvlecl commented on pypowsybl/network.py:
```diff
@@ -3984,6 +3997,36 @@ def create_curve_reactive_limits(self, df: _DataFrame, **kwargs: _ArrayLike) ->
+
+        Examples:
+
+            .. code-block:: python
```
I think it will not render correctly without a blank line

sylvlecl replied:
We need to add the methods to the API reference in sphinx doc

sylvlecl committed:
Minor adjustements:
- rename create_aliases to add_aliases for less confusion with elements creation
- fix typos in doc
- add methods to API reference doc
- empty type interpreted as "no type"

Signed-off-by: Sylvain Leclerc <sylvain.leclerc@rte-france.com>
